2016/05/29 21:49:52 [error] [exception.CHttpException.403] exception 'CHttpException' with message 'You are not authorized to perform this action.' in C:\xampp\htdocs\web\framework\web\auth\CAccessControlFilter.php:182
Stack trace:
#0 C:\xampp\htdocs\web\framework\web\auth\CAccessControlFilter.php(145): CAccessControlFilter->accessDenied(Object(CWebUser), 'You are not aut...')
#1 C:\xampp\htdocs\web\framework\web\filters\CFilter.php(38): CAccessControlFilter->preFilter(Object(CFilterChain))
#2 C:\xampp\htdocs\web\framework\web\CController.php(1145): CFilter->filter(Object(CFilterChain))
#3 C:\xampp\htdocs\web\framework\web\filters\CInlineFilter.php(58): CController->filterAccessControl(Object(CFilterChain))
#4 C:\xampp\htdocs\web\framework\web\filters\CFilterChain.php(130): CInlineFilter->filter(Object(CFilterChain))
#5 C:\xampp\htdocs\web\framework\web\CController.php(291): CFilterChain->run()
#6 C:\xampp\htdocs\web\framework\web\CController.php(265): CController->runActionWithFilters(Object(CInlineAction), Array)
#7 C:\xampp\htdocs\web\framework\web\CWebApplication.php(282): CController->run('index')
#8 C:\xampp\htdocs\web\framework\web\CWebApplication.php(141): CWebApplication->runController('calendarioForm/...')
#9 C:\xampp\htdocs\web\framework\base\CApplication.php(185): CWebApplication->processRequest()
#10 C:\xampp\htdocs\web\index.php(13): CApplication->run()
#11 {main}
REQUEST_URI=/web/calendarioForm/index
---
2016/05/29 21:49:59 [error] [exception.CHttpException.403] exception 'CHttpException' with message 'You are not authorized to perform this action.' in C:\xampp\htdocs\web\framework\web\auth\CAccessControlFilter.php:182
Stack trace:
#0 C:\xampp\htdocs\web\framework\web\auth\CAccessControlFilter.php(145): CAccessControlFilter->accessDenied(Object(CWebUser), 'You are not aut...')
#1 C:\xampp\htdocs\web\framework\web\filters\CFilter.php(38): CAccessControlFilter->preFilter(Object(CFilterChain))
#2 C:\xampp\htdocs\web\framework\web\CController.php(1145): CFilter->filter(Object(CFilterChain))
#3 C:\xampp\htdocs\web\framework\web\filters\CInlineFilter.php(58): CController->filterAccessControl(Object(CFilterChain))
#4 C:\xampp\htdocs\web\framework\web\filters\CFilterChain.php(130): CInlineFilter->filter(Object(CFilterChain))
#5 C:\xampp\htdocs\web\framework\web\CController.php(291): CFilterChain->run()
#6 C:\xampp\htdocs\web\framework\web\CController.php(265): CController->runActionWithFilters(Object(CInlineAction), Array)
#7 C:\xampp\htdocs\web\framework\web\CWebApplication.php(282): CController->run('admin')
#8 C:\xampp\htdocs\web\framework\web\CWebApplication.php(141): CWebApplication->runController('calendarioForm/...')
#9 C:\xampp\htdocs\web\framework\base\CApplication.php(185): CWebApplication->processRequest()
#10 C:\xampp\htdocs\web\index.php(13): CApplication->run()
#11 {main}
REQUEST_URI=/web/calendarioForm/admin
---
2016/05/29 21:50:01 [error] [exception.CHttpException.403] exception 'CHttpException' with message 'You are not authorized to perform this action.' in C:\xampp\htdocs\web\framework\web\auth\CAccessControlFilter.php:182
Stack trace:
#0 C:\xampp\htdocs\web\framework\web\auth\CAccessControlFilter.php(145): CAccessControlFilter->accessDenied(Object(CWebUser), 'You are not aut...')
#1 C:\xampp\htdocs\web\framework\web\filters\CFilter.php(38): CAccessControlFilter->preFilter(Object(CFilterChain))
#2 C:\xampp\htdocs\web\framework\web\CController.php(1145): CFilter->filter(Object(CFilterChain))
#3 C:\xampp\htdocs\web\framework\web\filters\CInlineFilter.php(58): CController->filterAccessControl(Object(CFilterChain))
#4 C:\xampp\htdocs\web\framework\web\filters\CFilterChain.php(130): CInlineFilter->filter(Object(CFilterChain))
#5 C:\xampp\htdocs\web\framework\web\CController.php(291): CFilterChain->run()
#6 C:\xampp\htdocs\web\framework\web\CController.php(265): CController->runActionWithFilters(Object(CInlineAction), Array)
#7 C:\xampp\htdocs\web\framework\web\CWebApplication.php(282): CController->run('index')
#8 C:\xampp\htdocs\web\framework\web\CWebApplication.php(141): CWebApplication->runController('calendarioForm/...')
#9 C:\xampp\htdocs\web\framework\base\CApplication.php(185): CWebApplication->processRequest()
#10 C:\xampp\htdocs\web\index.php(13): CApplication->run()
#11 {main}
REQUEST_URI=/web/calendarioForm/index
---
2016/05/30 00:04:14 [warning] [application] Failed to set unsafe attribute "repetir_password" of "RegistroForm".
in C:\xampp\htdocs\web\protected\controllers\RegistroFormController.php (75)
in C:\xampp\htdocs\web\index.php (13)
2016/05/30 00:04:39 [warning] [application] Failed to set unsafe attribute "repetir_password" of "RegistroForm".
in C:\xampp\htdocs\web\protected\controllers\RegistroFormController.php (75)
in C:\xampp\htdocs\web\index.php (13)
2016/05/30 00:06:53 [warning] [application] Failed to set unsafe attribute "repetir_password" of "RegistroForm".
in C:\xampp\htdocs\web\protected\controllers\RegistroFormController.php (75)
in C:\xampp\htdocs\web\index.php (13)
2016/05/30 00:07:31 [warning] [application] Failed to set unsafe attribute "repetir_password" of "RegistroForm".
in C:\xampp\htdocs\web\protected\controllers\RegistroFormController.php (75)
in C:\xampp\htdocs\web\index.php (13)
2016/05/30 00:09:27 [warning] [application] Failed to set unsafe attribute "repetir_password" of "RegistroForm".
in C:\xampp\htdocs\web\protected\controllers\RegistroFormController.php (75)
in C:\xampp\htdocs\web\index.php (13)
2016/05/30 00:12:11 [warning] [application] Failed to set unsafe attribute "repetir_password" of "RegistroForm".
in C:\xampp\htdocs\web\protected\controllers\RegistroFormController.php (75)
in C:\xampp\htdocs\web\index.php (13)
2016/05/30 22:12:17 [error] [exception.CHttpException.403] exception 'CHttpException' with message 'You are not authorized to perform this action.' in C:\xampp\htdocs\web\framework\web\auth\CAccessControlFilter.php:182
Stack trace:
#0 C:\xampp\htdocs\web\framework\web\auth\CAccessControlFilter.php(145): CAccessControlFilter->accessDenied(Object(CWebUser), 'You are not aut...')
#1 C:\xampp\htdocs\web\framework\web\filters\CFilter.php(38): CAccessControlFilter->preFilter(Object(CFilterChain))
#2 C:\xampp\htdocs\web\framework\web\CController.php(1145): CFilter->filter(Object(CFilterChain))
#3 C:\xampp\htdocs\web\framework\web\filters\CInlineFilter.php(58): CController->filterAccessControl(Object(CFilterChain))
#4 C:\xampp\htdocs\web\framework\web\filters\CFilterChain.php(130): CInlineFilter->filter(Object(CFilterChain))
#5 C:\xampp\htdocs\web\framework\web\CController.php(291): CFilterChain->run()
#6 C:\xampp\htdocs\web\framework\web\CController.php(265): CController->runActionWithFilters(Object(CInlineAction), Array)
#7 C:\xampp\htdocs\web\framework\web\CWebApplication.php(282): CController->run('create')
#8 C:\xampp\htdocs\web\framework\web\CWebApplication.php(141): CWebApplication->runController('cliente/create')
#9 C:\xampp\htdocs\web\framework\base\CApplication.php(185): CWebApplication->processRequest()
#10 C:\xampp\htdocs\web\index.php(13): CApplication->run()
#11 {main}
REQUEST_URI=/web/cliente/create
---
2016/05/30 22:15:00 [error] [exception.CHttpException.403] exception 'CHttpException' with message 'You are not authorized to perform this action.' in C:\xampp\htdocs\web\framework\web\auth\CAccessControlFilter.php:182
Stack trace:
#0 C:\xampp\htdocs\web\framework\web\auth\CAccessControlFilter.php(145): CAccessControlFilter->accessDenied(Object(CWebUser), 'You are not aut...')
#1 C:\xampp\htdocs\web\framework\web\filters\CFilter.php(38): CAccessControlFilter->preFilter(Object(CFilterChain))
#2 C:\xampp\htdocs\web\framework\web\CController.php(1145): CFilter->filter(Object(CFilterChain))
#3 C:\xampp\htdocs\web\framework\web\filters\CInlineFilter.php(58): CController->filterAccessControl(Object(CFilterChain))
#4 C:\xampp\htdocs\web\framework\web\filters\CFilterChain.php(130): CInlineFilter->filter(Object(CFilterChain))
#5 C:\xampp\htdocs\web\framework\web\CController.php(291): CFilterChain->run()
#6 C:\xampp\htdocs\web\framework\web\CController.php(265): CController->runActionWithFilters(Object(CInlineAction), Array)
#7 C:\xampp\htdocs\web\framework\web\CWebApplication.php(282): CController->run('')
#8 C:\xampp\htdocs\web\framework\web\CWebApplication.php(141): CWebApplication->runController('mano')
#9 C:\xampp\htdocs\web\framework\base\CApplication.php(185): CWebApplication->processRequest()
#10 C:\xampp\htdocs\web\index.php(13): CApplication->run()
#11 {main}
REQUEST_URI=/web/mano
---
2016/05/30 22:15:00 [error] [exception.CHttpException.403] exception 'CHttpException' with message 'You are not authorized to perform this action.' in C:\xampp\htdocs\web\framework\web\auth\CAccessControlFilter.php:182
Stack trace:
#0 C:\xampp\htdocs\web\framework\web\auth\CAccessControlFilter.php(145): CAccessControlFilter->accessDenied(Object(CWebUser), 'You are not aut...')
#1 C:\xampp\htdocs\web\framework\web\filters\CFilter.php(38): CAccessControlFilter->preFilter(Object(CFilterChain))
#2 C:\xampp\htdocs\web\framework\web\CController.php(1145): CFilter->filter(Object(CFilterChain))
#3 C:\xampp\htdocs\web\framework\web\filters\CInlineFilter.php(58): CController->filterAccessControl(Object(CFilterChain))
#4 C:\xampp\htdocs\web\framework\web\filters\CFilterChain.php(130): CInlineFilter->filter(Object(CFilterChain))
#5 C:\xampp\htdocs\web\framework\web\CController.php(291): CFilterChain->run()
#6 C:\xampp\htdocs\web\framework\web\CController.php(265): CController->runActionWithFilters(Object(CInlineAction), Array)
#7 C:\xampp\htdocs\web\framework\web\CWebApplication.php(282): CController->run('')
#8 C:\xampp\htdocs\web\framework\web\CWebApplication.php(141): CWebApplication->runController('pieza')
#9 C:\xampp\htdocs\web\framework\base\CApplication.php(185): CWebApplication->processRequest()
#10 C:\xampp\htdocs\web\index.php(13): CApplication->run()
#11 {main}
REQUEST_URI=/web/pieza
---
2016/05/30 22:15:01 [error] [exception.CHttpException.403] exception 'CHttpException' with message 'You are not authorized to perform this action.' in C:\xampp\htdocs\web\framework\web\auth\CAccessControlFilter.php:182
Stack trace:
#0 C:\xampp\htdocs\web\framework\web\auth\CAccessControlFilter.php(145): CAccessControlFilter->accessDenied(Object(CWebUser), 'You are not aut...')
#1 C:\xampp\htdocs\web\framework\web\filters\CFilter.php(38): CAccessControlFilter->preFilter(Object(CFilterChain))
#2 C:\xampp\htdocs\web\framework\web\CController.php(1145): CFilter->filter(Object(CFilterChain))
#3 C:\xampp\htdocs\web\framework\web\filters\CInlineFilter.php(58): CController->filterAccessControl(Object(CFilterChain))
#4 C:\xampp\htdocs\web\framework\web\filters\CFilterChain.php(130): CInlineFilter->filter(Object(CFilterChain))
#5 C:\xampp\htdocs\web\framework\web\CController.php(291): CFilterChain->run()
#6 C:\xampp\htdocs\web\framework\web\CController.php(265): CController->runActionWithFilters(Object(CInlineAction), Array)
#7 C:\xampp\htdocs\web\framework\web\CWebApplication.php(282): CController->run('')
#8 C:\xampp\htdocs\web\framework\web\CWebApplication.php(141): CWebApplication->runController('pieza')
#9 C:\xampp\htdocs\web\framework\base\CApplication.php(185): CWebApplication->processRequest()
#10 C:\xampp\htdocs\web\index.php(13): CApplication->run()
#11 {main}
REQUEST_URI=/web/pieza
---
2016/05/30 22:15:02 [error] [exception.CHttpException.403] exception 'CHttpException' with message 'You are not authorized to perform this action.' in C:\xampp\htdocs\web\framework\web\auth\CAccessControlFilter.php:182
Stack trace:
#0 C:\xampp\htdocs\web\framework\web\auth\CAccessControlFilter.php(145): CAccessControlFilter->accessDenied(Object(CWebUser), 'You are not aut...')
#1 C:\xampp\htdocs\web\framework\web\filters\CFilter.php(38): CAccessControlFilter->preFilter(Object(CFilterChain))
#2 C:\xampp\htdocs\web\framework\web\CController.php(1145): CFilter->filter(Object(CFilterChain))
#3 C:\xampp\htdocs\web\framework\web\filters\CInlineFilter.php(58): CController->filterAccessControl(Object(CFilterChain))
#4 C:\xampp\htdocs\web\framework\web\filters\CFilterChain.php(130): CInlineFilter->filter(Object(CFilterChain))
#5 C:\xampp\htdocs\web\framework\web\CController.php(291): CFilterChain->run()
#6 C:\xampp\htdocs\web\framework\web\CController.php(265): CController->runActionWithFilters(Object(CInlineAction), Array)
#7 C:\xampp\htdocs\web\framework\web\CWebApplication.php(282): CController->run('')
#8 C:\xampp\htdocs\web\framework\web\CWebApplication.php(141): CWebApplication->runController('estado')
#9 C:\xampp\htdocs\web\framework\base\CApplication.php(185): CWebApplication->processRequest()
#10 C:\xampp\htdocs\web\index.php(13): CApplication->run()
#11 {main}
REQUEST_URI=/web/estado
---
2016/05/30 22:15:03 [error] [exception.CHttpException.403] exception 'CHttpException' with message 'You are not authorized to perform this action.' in C:\xampp\htdocs\web\framework\web\auth\CAccessControlFilter.php:182
Stack trace:
#0 C:\xampp\htdocs\web\framework\web\auth\CAccessControlFilter.php(145): CAccessControlFilter->accessDenied(Object(CWebUser), 'You are not aut...')
#1 C:\xampp\htdocs\web\framework\web\filters\CFilter.php(38): CAccessControlFilter->preFilter(Object(CFilterChain))
#2 C:\xampp\htdocs\web\framework\web\CController.php(1145): CFilter->filter(Object(CFilterChain))
#3 C:\xampp\htdocs\web\framework\web\filters\CInlineFilter.php(58): CController->filterAccessControl(Object(CFilterChain))
#4 C:\xampp\htdocs\web\framework\web\filters\CFilterChain.php(130): CInlineFilter->filter(Object(CFilterChain))
#5 C:\xampp\htdocs\web\framework\web\CController.php(291): CFilterChain->run()
#6 C:\xampp\htdocs\web\framework\web\CController.php(265): CController->runActionWithFilters(Object(CInlineAction), Array)
#7 C:\xampp\htdocs\web\framework\web\CWebApplication.php(282): CController->run('create')
#8 C:\xampp\htdocs\web\framework\web\CWebApplication.php(141): CWebApplication->runController('handicap/create')
#9 C:\xampp\htdocs\web\framework\base\CApplication.php(185): CWebApplication->processRequest()
#10 C:\xampp\htdocs\web\index.php(13): CApplication->run()
#11 {main}
REQUEST_URI=/web/handicap/create
HTTP_REFERER=http://localhost/web/handicap
---
2016/05/30 22:15:04 [error] [exception.CHttpException.403] exception 'CHttpException' with message 'You are not authorized to perform this action.' in C:\xampp\htdocs\web\framework\web\auth\CAccessControlFilter.php:182
Stack trace:
#0 C:\xampp\htdocs\web\framework\web\auth\CAccessControlFilter.php(145): CAccessControlFilter->accessDenied(Object(CWebUser), 'You are not aut...')
#1 C:\xampp\htdocs\web\framework\web\filters\CFilter.php(38): CAccessControlFilter->preFilter(Object(CFilterChain))
#2 C:\xampp\htdocs\web\framework\web\CController.php(1145): CFilter->filter(Object(CFilterChain))
#3 C:\xampp\htdocs\web\framework\web\filters\CInlineFilter.php(58): CController->filterAccessControl(Object(CFilterChain))
#4 C:\xampp\htdocs\web\framework\web\filters\CFilterChain.php(130): CInlineFilter->filter(Object(CFilterChain))
#5 C:\xampp\htdocs\web\framework\web\CController.php(291): CFilterChain->run()
#6 C:\xampp\htdocs\web\framework\web\CController.php(265): CController->runActionWithFilters(Object(CInlineAction), Array)
#7 C:\xampp\htdocs\web\framework\web\CWebApplication.php(282): CController->run('create')
#8 C:\xampp\htdocs\web\framework\web\CWebApplication.php(141): CWebApplication->runController('handicap/create')
#9 C:\xampp\htdocs\web\framework\base\CApplication.php(185): CWebApplication->processRequest()
#10 C:\xampp\htdocs\web\index.php(13): CApplication->run()
#11 {main}
REQUEST_URI=/web/handicap/create
HTTP_REFERER=http://localhost/web/handicap
---
2016/05/30 22:15:07 [error] [exception.CHttpException.403] exception 'CHttpException' with message 'You are not authorized to perform this action.' in C:\xampp\htdocs\web\framework\web\auth\CAccessControlFilter.php:182
Stack trace:
#0 C:\xampp\htdocs\web\framework\web\auth\CAccessControlFilter.php(145): CAccessControlFilter->accessDenied(Object(CWebUser), 'You are not aut...')
#1 C:\xampp\htdocs\web\framework\web\filters\CFilter.php(38): CAccessControlFilter->preFilter(Object(CFilterChain))
#2 C:\xampp\htdocs\web\framework\web\CController.php(1145): CFilter->filter(Object(CFilterChain))
#3 C:\xampp\htdocs\web\framework\web\filters\CInlineFilter.php(58): CController->filterAccessControl(Object(CFilterChain))
#4 C:\xampp\htdocs\web\framework\web\filters\CFilterChain.php(130): CInlineFilter->filter(Object(CFilterChain))
#5 C:\xampp\htdocs\web\framework\web\CController.php(291): CFilterChain->run()
#6 C:\xampp\htdocs\web\framework\web\CController.php(265): CController->runActionWithFilters(Object(CInlineAction), Array)
#7 C:\xampp\htdocs\web\framework\web\CWebApplication.php(282): CController->run('')
#8 C:\xampp\htdocs\web\framework\web\CWebApplication.php(141): CWebApplication->runController('handicap')
#9 C:\xampp\htdocs\web\framework\base\CApplication.php(185): CWebApplication->processRequest()
#10 C:\xampp\htdocs\web\index.php(13): CApplication->run()
#11 {main}
REQUEST_URI=/web/handicap
---
2016/05/30 22:15:09 [error] [exception.CHttpException.403] exception 'CHttpException' with message 'You are not authorized to perform this action.' in C:\xampp\htdocs\web\framework\web\auth\CAccessControlFilter.php:182
Stack trace:
#0 C:\xampp\htdocs\web\framework\web\auth\CAccessControlFilter.php(145): CAccessControlFilter->accessDenied(Object(CWebUser), 'You are not aut...')
#1 C:\xampp\htdocs\web\framework\web\filters\CFilter.php(38): CAccessControlFilter->preFilter(Object(CFilterChain))
#2 C:\xampp\htdocs\web\framework\web\CController.php(1145): CFilter->filter(Object(CFilterChain))
#3 C:\xampp\htdocs\web\framework\web\filters\CInlineFilter.php(58): CController->filterAccessControl(Object(CFilterChain))
#4 C:\xampp\htdocs\web\framework\web\filters\CFilterChain.php(130): CInlineFilter->filter(Object(CFilterChain))
#5 C:\xampp\htdocs\web\framework\web\CController.php(291): CFilterChain->run()
#6 C:\xampp\htdocs\web\framework\web\CController.php(265): CController->runActionWithFilters(Object(CInlineAction), Array)
#7 C:\xampp\htdocs\web\framework\web\CWebApplication.php(282): CController->run('create')
#8 C:\xampp\htdocs\web\framework\web\CWebApplication.php(141): CWebApplication->runController('handicap/create')
#9 C:\xampp\htdocs\web\framework\base\CApplication.php(185): CWebApplication->processRequest()
#10 C:\xampp\htdocs\web\index.php(13): CApplication->run()
#11 {main}
REQUEST_URI=/web/handicap/create
HTTP_REFERER=http://localhost/web/handicap
---
2016/05/30 22:15:09 [error] [exception.CHttpException.403] exception 'CHttpException' with message 'You are not authorized to perform this action.' in C:\xampp\htdocs\web\framework\web\auth\CAccessControlFilter.php:182
Stack trace:
#0 C:\xampp\htdocs\web\framework\web\auth\CAccessControlFilter.php(145): CAccessControlFilter->accessDenied(Object(CWebUser), 'You are not aut...')
#1 C:\xampp\htdocs\web\framework\web\filters\CFilter.php(38): CAccessControlFilter->preFilter(Object(CFilterChain))
#2 C:\xampp\htdocs\web\framework\web\CController.php(1145): CFilter->filter(Object(CFilterChain))
#3 C:\xampp\htdocs\web\framework\web\filters\CInlineFilter.php(58): CController->filterAccessControl(Object(CFilterChain))
#4 C:\xampp\htdocs\web\framework\web\filters\CFilterChain.php(130): CInlineFilter->filter(Object(CFilterChain))
#5 C:\xampp\htdocs\web\framework\web\CController.php(291): CFilterChain->run()
#6 C:\xampp\htdocs\web\framework\web\CController.php(265): CController->runActionWithFilters(Object(CInlineAction), Array)
#7 C:\xampp\htdocs\web\framework\web\CWebApplication.php(282): CController->run('')
#8 C:\xampp\htdocs\web\framework\web\CWebApplication.php(141): CWebApplication->runController('handicap')
#9 C:\xampp\htdocs\web\framework\base\CApplication.php(185): CWebApplication->processRequest()
#10 C:\xampp\htdocs\web\index.php(13): CApplication->run()
#11 {main}
REQUEST_URI=/web/handicap
---
2016/05/30 22:15:10 [error] [exception.CHttpException.403] exception 'CHttpException' with message 'You are not authorized to perform this action.' in C:\xampp\htdocs\web\framework\web\auth\CAccessControlFilter.php:182
Stack trace:
#0 C:\xampp\htdocs\web\framework\web\auth\CAccessControlFilter.php(145): CAccessControlFilter->accessDenied(Object(CWebUser), 'You are not aut...')
#1 C:\xampp\htdocs\web\framework\web\filters\CFilter.php(38): CAccessControlFilter->preFilter(Object(CFilterChain))
#2 C:\xampp\htdocs\web\framework\web\CController.php(1145): CFilter->filter(Object(CFilterChain))
#3 C:\xampp\htdocs\web\framework\web\filters\CInlineFilter.php(58): CController->filterAccessControl(Object(CFilterChain))
#4 C:\xampp\htdocs\web\framework\web\filters\CFilterChain.php(130): CInlineFilter->filter(Object(CFilterChain))
#5 C:\xampp\htdocs\web\framework\web\CController.php(291): CFilterChain->run()
#6 C:\xampp\htdocs\web\framework\web\CController.php(265): CController->runActionWithFilters(Object(CInlineAction), Array)
#7 C:\xampp\htdocs\web\framework\web\CWebApplication.php(282): CController->run('create')
#8 C:\xampp\htdocs\web\framework\web\CWebApplication.php(141): CWebApplication->runController('cliente/create')
#9 C:\xampp\htdocs\web\framework\base\CApplication.php(185): CWebApplication->processRequest()
#10 C:\xampp\htdocs\web\index.php(13): CApplication->run()
#11 {main}
REQUEST_URI=/web/cliente/create
---
2016/05/30 22:15:12 [error] [exception.CHttpException.403] exception 'CHttpException' with message 'You are not authorized to perform this action.' in C:\xampp\htdocs\web\framework\web\auth\CAccessControlFilter.php:182
Stack trace:
#0 C:\xampp\htdocs\web\framework\web\auth\CAccessControlFilter.php(145): CAccessControlFilter->accessDenied(Object(CWebUser), 'You are not aut...')
#1 C:\xampp\htdocs\web\framework\web\filters\CFilter.php(38): CAccessControlFilter->preFilter(Object(CFilterChain))
#2 C:\xampp\htdocs\web\framework\web\CController.php(1145): CFilter->filter(Object(CFilterChain))
#3 C:\xampp\htdocs\web\framework\web\filters\CInlineFilter.php(58): CController->filterAccessControl(Object(CFilterChain))
#4 C:\xampp\htdocs\web\framework\web\filters\CFilterChain.php(130): CInlineFilter->filter(Object(CFilterChain))
#5 C:\xampp\htdocs\web\framework\web\CController.php(291): CFilterChain->run()
#6 C:\xampp\htdocs\web\framework\web\CController.php(265): CController->runActionWithFilters(Object(CInlineAction), Array)
#7 C:\xampp\htdocs\web\framework\web\CWebApplication.php(282): CController->run('create')
#8 C:\xampp\htdocs\web\framework\web\CWebApplication.php(141): CWebApplication->runController('cliente/create')
#9 C:\xampp\htdocs\web\framework\base\CApplication.php(185): CWebApplication->processRequest()
#10 C:\xampp\htdocs\web\index.php(13): CApplication->run()
#11 {main}
REQUEST_URI=/web/cliente/create
---
2016/05/30 22:38:45 [error] [system.db.CDbCommand] CDbCommand::execute() failed: SQLSTATE[23000]: Integrity constraint violation: 1452 Cannot add or update a child row: a foreign key constraint fails (`web`.`pieza`, CONSTRAINT `fk_pieza_marca_pieza1` FOREIGN KEY (`marca_pieza_id`) REFERENCES `marca_pieza` (`id`) ON DELETE NO ACTION ON UPDATE NO ACTION). The SQL statement executed was: INSERT INTO `pieza` (`vivo`, `nombre`, `precio`, `precio_instalar`, `fecha_creacion`, `fecha_modificacion`, `marca_pieza_id`, `imagen`, `observaciones`, `minutos_instalacion`, `informacion`) VALUES (:yp0, :yp1, :yp2, :yp3, :yp4, :yp5, :yp6, :yp7, :yp8, :yp9, :yp10).
in C:\xampp\htdocs\web\protected\controllers\PiezaController.php (75)
in C:\xampp\htdocs\web\index.php (13)
2016/05/30 22:38:45 [error] [exception.CDbException] exception 'CDbException' with message 'CDbCommand failed to execute the SQL statement: SQLSTATE[23000]: Integrity constraint violation: 1452 Cannot add or update a child row: a foreign key constraint fails (`web`.`pieza`, CONSTRAINT `fk_pieza_marca_pieza1` FOREIGN KEY (`marca_pieza_id`) REFERENCES `marca_pieza` (`id`) ON DELETE NO ACTION ON UPDATE NO ACTION). The SQL statement executed was: INSERT INTO `pieza` (`vivo`, `nombre`, `precio`, `precio_instalar`, `fecha_creacion`, `fecha_modificacion`, `marca_pieza_id`, `imagen`, `observaciones`, `minutos_instalacion`, `informacion`) VALUES (:yp0, :yp1, :yp2, :yp3, :yp4, :yp5, :yp6, :yp7, :yp8, :yp9, :yp10)' in C:\xampp\htdocs\web\framework\db\CDbCommand.php:358
Stack trace:
#0 C:\xampp\htdocs\web\framework\db\ar\CActiveRecord.php(1080): CDbCommand->execute()
#1 C:\xampp\htdocs\web\framework\db\ar\CActiveRecord.php(809): CActiveRecord->insert(NULL)
#2 C:\xampp\htdocs\web\protected\controllers\PiezaController.php(75): CActiveRecord->save()
#3 C:\xampp\htdocs\web\framework\web\actions\CInlineAction.php(49): PiezaController->actionCreate()
#4 C:\xampp\htdocs\web\framework\web\CController.php(308): CInlineAction->runWithParams(Array)
#5 C:\xampp\htdocs\web\framework\web\filters\CFilterChain.php(133): CController->runAction(Object(CInlineAction))
#6 C:\xampp\htdocs\web\framework\web\filters\CFilter.php(40): CFilterChain->run()
#7 C:\xampp\htdocs\web\framework\web\CController.php(1145): CFilter->filter(Object(CFilterChain))
#8 C:\xampp\htdocs\web\framework\web\filters\CInlineFilter.php(58): CController->filterAccessControl(Object(CFilterChain))
#9 C:\xampp\htdocs\web\framework\web\filters\CFilterChain.php(130): CInlineFilter->filter(Object(CFilterChain))
#10 C:\xampp\htdocs\web\framework\web\CController.php(291): CFilterChain->run()
#11 C:\xampp\htdocs\web\framework\web\CController.php(265): CController->runActionWithFilters(Object(CInlineAction), Array)
#12 C:\xampp\htdocs\web\framework\web\CWebApplication.php(282): CController->run('create')
#13 C:\xampp\htdocs\web\framework\web\CWebApplication.php(141): CWebApplication->runController('pieza/create')
#14 C:\xampp\htdocs\web\framework\base\CApplication.php(185): CWebApplication->processRequest()
#15 C:\xampp\htdocs\web\index.php(13): CApplication->run()
#16 {main}
REQUEST_URI=/web/pieza/create
HTTP_REFERER=http://localhost/web/pieza/create
---
2016/05/30 22:40:44 [error] [system.db.CDbCommand] CDbCommand::execute() failed: SQLSTATE[23000]: Integrity constraint violation: 1452 Cannot add or update a child row: a foreign key constraint fails (`web`.`pieza`, CONSTRAINT `fk_pieza_marca_pieza1` FOREIGN KEY (`marca_pieza_id`) REFERENCES `marca_pieza` (`id`) ON DELETE NO ACTION ON UPDATE NO ACTION). The SQL statement executed was: INSERT INTO `pieza` (`vivo`, `nombre`, `precio`, `precio_instalar`, `fecha_creacion`, `fecha_modificacion`, `marca_pieza_id`, `imagen`, `observaciones`, `minutos_instalacion`, `informacion`) VALUES (:yp0, :yp1, :yp2, :yp3, :yp4, :yp5, :yp6, :yp7, :yp8, :yp9, :yp10).
in C:\xampp\htdocs\web\protected\controllers\PiezaController.php (75)
in C:\xampp\htdocs\web\index.php (13)
2016/05/30 22:40:44 [error] [exception.CDbException] exception 'CDbException' with message 'CDbCommand failed to execute the SQL statement: SQLSTATE[23000]: Integrity constraint violation: 1452 Cannot add or update a child row: a foreign key constraint fails (`web`.`pieza`, CONSTRAINT `fk_pieza_marca_pieza1` FOREIGN KEY (`marca_pieza_id`) REFERENCES `marca_pieza` (`id`) ON DELETE NO ACTION ON UPDATE NO ACTION). The SQL statement executed was: INSERT INTO `pieza` (`vivo`, `nombre`, `precio`, `precio_instalar`, `fecha_creacion`, `fecha_modificacion`, `marca_pieza_id`, `imagen`, `observaciones`, `minutos_instalacion`, `informacion`) VALUES (:yp0, :yp1, :yp2, :yp3, :yp4, :yp5, :yp6, :yp7, :yp8, :yp9, :yp10)' in C:\xampp\htdocs\web\framework\db\CDbCommand.php:358
Stack trace:
#0 C:\xampp\htdocs\web\framework\db\ar\CActiveRecord.php(1080): CDbCommand->execute()
#1 C:\xampp\htdocs\web\framework\db\ar\CActiveRecord.php(809): CActiveRecord->insert(NULL)
#2 C:\xampp\htdocs\web\protected\controllers\PiezaController.php(75): CActiveRecord->save()
#3 C:\xampp\htdocs\web\framework\web\actions\CInlineAction.php(49): PiezaController->actionCreate()
#4 C:\xampp\htdocs\web\framework\web\CController.php(308): CInlineAction->runWithParams(Array)
#5 C:\xampp\htdocs\web\framework\web\filters\CFilterChain.php(133): CController->runAction(Object(CInlineAction))
#6 C:\xampp\htdocs\web\framework\web\filters\CFilter.php(40): CFilterChain->run()
#7 C:\xampp\htdocs\web\framework\web\CController.php(1145): CFilter->filter(Object(CFilterChain))
#8 C:\xampp\htdocs\web\framework\web\filters\CInlineFilter.php(58): CController->filterAccessControl(Object(CFilterChain))
#9 C:\xampp\htdocs\web\framework\web\filters\CFilterChain.php(130): CInlineFilter->filter(Object(CFilterChain))
#10 C:\xampp\htdocs\web\framework\web\CController.php(291): CFilterChain->run()
#11 C:\xampp\htdocs\web\framework\web\CController.php(265): CController->runActionWithFilters(Object(CInlineAction), Array)
#12 C:\xampp\htdocs\web\framework\web\CWebApplication.php(282): CController->run('create')
#13 C:\xampp\htdocs\web\framework\web\CWebApplication.php(141): CWebApplication->runController('pieza/create')
#14 C:\xampp\htdocs\web\framework\base\CApplication.php(185): CWebApplication->processRequest()
#15 C:\xampp\htdocs\web\index.php(13): CApplication->run()
#16 {main}
REQUEST_URI=/web/pieza/create
HTTP_REFERER=http://localhost/web/pieza/create
---
2016/05/30 22:41:57 [error] [system.db.CDbCommand] CDbCommand::execute() failed: SQLSTATE[23000]: Integrity constraint violation: 1452 Cannot add or update a child row: a foreign key constraint fails (`web`.`pieza`, CONSTRAINT `fk_pieza_marca_pieza1` FOREIGN KEY (`marca_pieza_id`) REFERENCES `marca_pieza` (`id`) ON DELETE NO ACTION ON UPDATE NO ACTION). The SQL statement executed was: INSERT INTO `pieza` (`vivo`, `nombre`, `precio`, `precio_instalar`, `fecha_creacion`, `fecha_modificacion`, `marca_pieza_id`, `imagen`, `observaciones`, `minutos_instalacion`, `informacion`) VALUES (:yp0, :yp1, :yp2, :yp3, :yp4, :yp5, :yp6, :yp7, :yp8, :yp9, :yp10).
in C:\xampp\htdocs\web\protected\controllers\PiezaController.php (75)
in C:\xampp\htdocs\web\index.php (13)
2016/05/30 22:41:57 [error] [exception.CDbException] exception 'CDbException' with message 'CDbCommand failed to execute the SQL statement: SQLSTATE[23000]: Integrity constraint violation: 1452 Cannot add or update a child row: a foreign key constraint fails (`web`.`pieza`, CONSTRAINT `fk_pieza_marca_pieza1` FOREIGN KEY (`marca_pieza_id`) REFERENCES `marca_pieza` (`id`) ON DELETE NO ACTION ON UPDATE NO ACTION). The SQL statement executed was: INSERT INTO `pieza` (`vivo`, `nombre`, `precio`, `precio_instalar`, `fecha_creacion`, `fecha_modificacion`, `marca_pieza_id`, `imagen`, `observaciones`, `minutos_instalacion`, `informacion`) VALUES (:yp0, :yp1, :yp2, :yp3, :yp4, :yp5, :yp6, :yp7, :yp8, :yp9, :yp10)' in C:\xampp\htdocs\web\framework\db\CDbCommand.php:358
Stack trace:
#0 C:\xampp\htdocs\web\framework\db\ar\CActiveRecord.php(1080): CDbCommand->execute()
#1 C:\xampp\htdocs\web\framework\db\ar\CActiveRecord.php(809): CActiveRecord->insert(NULL)
#2 C:\xampp\htdocs\web\protected\controllers\PiezaController.php(75): CActiveRecord->save()
#3 C:\xampp\htdocs\web\framework\web\actions\CInlineAction.php(49): PiezaController->actionCreate()
#4 C:\xampp\htdocs\web\framework\web\CController.php(308): CInlineAction->runWithParams(Array)
#5 C:\xampp\htdocs\web\framework\web\filters\CFilterChain.php(133): CController->runAction(Object(CInlineAction))
#6 C:\xampp\htdocs\web\framework\web\filters\CFilter.php(40): CFilterChain->run()
#7 C:\xampp\htdocs\web\framework\web\CController.php(1145): CFilter->filter(Object(CFilterChain))
#8 C:\xampp\htdocs\web\framework\web\filters\CInlineFilter.php(58): CController->filterAccessControl(Object(CFilterChain))
#9 C:\xampp\htdocs\web\framework\web\filters\CFilterChain.php(130): CInlineFilter->filter(Object(CFilterChain))
#10 C:\xampp\htdocs\web\framework\web\CController.php(291): CFilterChain->run()
#11 C:\xampp\htdocs\web\framework\web\CController.php(265): CController->runActionWithFilters(Object(CInlineAction), Array)
#12 C:\xampp\htdocs\web\framework\web\CWebApplication.php(282): CController->run('create')
#13 C:\xampp\htdocs\web\framework\web\CWebApplication.php(141): CWebApplication->runController('pieza/create')
#14 C:\xampp\htdocs\web\framework\base\CApplication.php(185): CWebApplication->processRequest()
#15 C:\xampp\htdocs\web\index.php(13): CApplication->run()
#16 {main}
REQUEST_URI=/web/pieza/create
HTTP_REFERER=http://localhost/web/pieza/create
---
2016/05/30 22:44:08 [error] [system.db.CDbCommand] CDbCommand::execute() failed: SQLSTATE[23000]: Integrity constraint violation: 1451 Cannot delete or update a parent row: a foreign key constraint fails (`web`.`pieza`, CONSTRAINT `fk_pieza_marca_pieza1` FOREIGN KEY (`marca_pieza_id`) REFERENCES `marca_pieza` (`id`) ON DELETE NO ACTION ON UPDATE NO ACTION). The SQL statement executed was: DELETE FROM `marca_pieza` WHERE `marca_pieza`.`id`=1.
in C:\xampp\htdocs\web\protected\controllers\MarcaPiezaController.php (115)
in C:\xampp\htdocs\web\index.php (13)
2016/05/30 22:44:08 [error] [exception.CDbException] exception 'CDbException' with message 'CDbCommand failed to execute the SQL statement: SQLSTATE[23000]: Integrity constraint violation: 1451 Cannot delete or update a parent row: a foreign key constraint fails (`web`.`pieza`, CONSTRAINT `fk_pieza_marca_pieza1` FOREIGN KEY (`marca_pieza_id`) REFERENCES `marca_pieza` (`id`) ON DELETE NO ACTION ON UPDATE NO ACTION). The SQL statement executed was: DELETE FROM `marca_pieza` WHERE `marca_pieza`.`id`=1' in C:\xampp\htdocs\web\framework\db\CDbCommand.php:358
Stack trace:
#0 C:\xampp\htdocs\web\framework\db\ar\CActiveRecord.php(1807): CDbCommand->execute()
#1 C:\xampp\htdocs\web\framework\db\ar\CActiveRecord.php(1227): CActiveRecord->deleteByPk('1')
#2 C:\xampp\htdocs\web\protected\controllers\MarcaPiezaController.php(115): CActiveRecord->delete()
#3 [internal function]: MarcaPiezaController->actionDelete('1')
#4 C:\xampp\htdocs\web\framework\web\actions\CAction.php(109): ReflectionMethod->invokeArgs(Object(MarcaPiezaController), Array)
#5 C:\xampp\htdocs\web\framework\web\actions\CInlineAction.php(47): CAction->runWithParamsInternal(Object(MarcaPiezaController), Object(ReflectionMethod), Array)
#6 C:\xampp\htdocs\web\framework\web\CController.php(308): CInlineAction->runWithParams(Array)
#7 C:\xampp\htdocs\web\framework\web\filters\CFilterChain.php(133): CController->runAction(Object(CInlineAction))
#8 C:\xampp\htdocs\web\framework\web\CController.php(1116): CFilterChain->run()
#9 C:\xampp\htdocs\web\framework\web\filters\CInlineFilter.php(58): CController->filterPostOnly(Object(CFilterChain))
#10 C:\xampp\htdocs\web\framework\web\filters\CFilterChain.php(130): CInlineFilter->filter(Object(CFilterChain))
#11 C:\xampp\htdocs\web\framework\web\filters\CFilter.php(40): CFilterChain->run()
#12 C:\xampp\htdocs\web\framework\web\CController.php(1145): CFilter->filter(Object(CFilterChain))
#13 C:\xampp\htdocs\web\framework\web\filters\CInlineFilter.php(58): CController->filterAccessControl(Object(CFilterChain))
#14 C:\xampp\htdocs\web\framework\web\filters\CFilterChain.php(130): CInlineFilter->filter(Object(CFilterChain))
#15 C:\xampp\htdocs\web\framework\web\CController.php(291): CFilterChain->run()
#16 C:\xampp\htdocs\web\framework\web\CController.php(265): CController->runActionWithFilters(Object(CInlineAction), Array)
#17 C:\xampp\htdocs\web\framework\web\CWebApplication.php(282): CController->run('delete')
#18 C:\xampp\htdocs\web\framework\web\CWebApplication.php(141): CWebApplication->runController('marcaPieza/dele...')
#19 C:\xampp\htdocs\web\framework\base\CApplication.php(185): CWebApplication->processRequest()
#20 C:\xampp\htdocs\web\index.php(13): CApplication->run()
#21 {main}
REQUEST_URI=/web/marcaPieza/delete/1?ajax=marca-pieza-grid
HTTP_REFERER=http://localhost/web/marcaPieza/admin
---
2016/05/30 22:44:27 [error] [system.db.CDbCommand] CDbCommand::execute() failed: SQLSTATE[23000]: Integrity constraint violation: 1451 Cannot delete or update a parent row: a foreign key constraint fails (`web`.`pieza`, CONSTRAINT `fk_pieza_marca_pieza1` FOREIGN KEY (`marca_pieza_id`) REFERENCES `marca_pieza` (`id`) ON DELETE NO ACTION ON UPDATE NO ACTION). The SQL statement executed was: DELETE FROM `marca_pieza` WHERE `marca_pieza`.`id`=1.
in C:\xampp\htdocs\web\protected\controllers\MarcaPiezaController.php (115)
in C:\xampp\htdocs\web\index.php (13)
2016/05/30 22:44:27 [error] [exception.CDbException] exception 'CDbException' with message 'CDbCommand failed to execute the SQL statement: SQLSTATE[23000]: Integrity constraint violation: 1451 Cannot delete or update a parent row: a foreign key constraint fails (`web`.`pieza`, CONSTRAINT `fk_pieza_marca_pieza1` FOREIGN KEY (`marca_pieza_id`) REFERENCES `marca_pieza` (`id`) ON DELETE NO ACTION ON UPDATE NO ACTION). The SQL statement executed was: DELETE FROM `marca_pieza` WHERE `marca_pieza`.`id`=1' in C:\xampp\htdocs\web\framework\db\CDbCommand.php:358
Stack trace:
#0 C:\xampp\htdocs\web\framework\db\ar\CActiveRecord.php(1807): CDbCommand->execute()
#1 C:\xampp\htdocs\web\framework\db\ar\CActiveRecord.php(1227): CActiveRecord->deleteByPk('1')
#2 C:\xampp\htdocs\web\protected\controllers\MarcaPiezaController.php(115): CActiveRecord->delete()
#3 [internal function]: MarcaPiezaController->actionDelete('1')
#4 C:\xampp\htdocs\web\framework\web\actions\CAction.php(109): ReflectionMethod->invokeArgs(Object(MarcaPiezaController), Array)
#5 C:\xampp\htdocs\web\framework\web\actions\CInlineAction.php(47): CAction->runWithParamsInternal(Object(MarcaPiezaController), Object(ReflectionMethod), Array)
#6 C:\xampp\htdocs\web\framework\web\CController.php(308): CInlineAction->runWithParams(Array)
#7 C:\xampp\htdocs\web\framework\web\filters\CFilterChain.php(133): CController->runAction(Object(CInlineAction))
#8 C:\xampp\htdocs\web\framework\web\CController.php(1116): CFilterChain->run()
#9 C:\xampp\htdocs\web\framework\web\filters\CInlineFilter.php(58): CController->filterPostOnly(Object(CFilterChain))
#10 C:\xampp\htdocs\web\framework\web\filters\CFilterChain.php(130): CInlineFilter->filter(Object(CFilterChain))
#11 C:\xampp\htdocs\web\framework\web\filters\CFilter.php(40): CFilterChain->run()
#12 C:\xampp\htdocs\web\framework\web\CController.php(1145): CFilter->filter(Object(CFilterChain))
#13 C:\xampp\htdocs\web\framework\web\filters\CInlineFilter.php(58): CController->filterAccessControl(Object(CFilterChain))
#14 C:\xampp\htdocs\web\framework\web\filters\CFilterChain.php(130): CInlineFilter->filter(Object(CFilterChain))
#15 C:\xampp\htdocs\web\framework\web\CController.php(291): CFilterChain->run()
#16 C:\xampp\htdocs\web\framework\web\CController.php(265): CController->runActionWithFilters(Object(CInlineAction), Array)
#17 C:\xampp\htdocs\web\framework\web\CWebApplication.php(282): CController->run('delete')
#18 C:\xampp\htdocs\web\framework\web\CWebApplication.php(141): CWebApplication->runController('marcaPieza/dele...')
#19 C:\xampp\htdocs\web\framework\base\CApplication.php(185): CWebApplication->processRequest()
#20 C:\xampp\htdocs\web\index.php(13): CApplication->run()
#21 {main}
REQUEST_URI=/web/marcaPieza/delete/1?ajax=marca-pieza-grid
HTTP_REFERER=http://localhost/web/marcaPieza/admin
---
2016/05/31 21:47:43 [error] [system.db.CDbCommand] CDbCommand::execute() failed: SQLSTATE[23000]: Integrity constraint violation: 1452 Cannot add or update a child row: a foreign key constraint fails (`web`.`handicap`, CONSTRAINT `fk_handicap_mano1` FOREIGN KEY (`mano_id`) REFERENCES `mano` (`id`) ON DELETE NO ACTION ON UPDATE NO ACTION). The SQL statement executed was: INSERT INTO `handicap` (`vivo`, `nombre`, `recargo`, `minutos_duracion`, `imagen`, `fecha_creacion`, `fecha_modificacion`, `observaciones`, `informacion`, `mano_id`) VALUES (:yp0, :yp1, :yp2, :yp3, :yp4, :yp5, :yp6, :yp7, :yp8, :yp9).
in C:\xampp\htdocs\web\protected\controllers\HandicapController.php (75)
in C:\xampp\htdocs\web\index.php (13)
2016/05/31 21:47:43 [error] [exception.CDbException] exception 'CDbException' with message 'CDbCommand failed to execute the SQL statement: SQLSTATE[23000]: Integrity constraint violation: 1452 Cannot add or update a child row: a foreign key constraint fails (`web`.`handicap`, CONSTRAINT `fk_handicap_mano1` FOREIGN KEY (`mano_id`) REFERENCES `mano` (`id`) ON DELETE NO ACTION ON UPDATE NO ACTION). The SQL statement executed was: INSERT INTO `handicap` (`vivo`, `nombre`, `recargo`, `minutos_duracion`, `imagen`, `fecha_creacion`, `fecha_modificacion`, `observaciones`, `informacion`, `mano_id`) VALUES (:yp0, :yp1, :yp2, :yp3, :yp4, :yp5, :yp6, :yp7, :yp8, :yp9)' in C:\xampp\htdocs\web\framework\db\CDbCommand.php:358
Stack trace:
#0 C:\xampp\htdocs\web\framework\db\ar\CActiveRecord.php(1080): CDbCommand->execute()
#1 C:\xampp\htdocs\web\framework\db\ar\CActiveRecord.php(809): CActiveRecord->insert(NULL)
#2 C:\xampp\htdocs\web\protected\controllers\HandicapController.php(75): CActiveRecord->save()
#3 C:\xampp\htdocs\web\framework\web\actions\CInlineAction.php(49): HandicapController->actionCreate()
#4 C:\xampp\htdocs\web\framework\web\CController.php(308): CInlineAction->runWithParams(Array)
#5 C:\xampp\htdocs\web\framework\web\filters\CFilterChain.php(133): CController->runAction(Object(CInlineAction))
#6 C:\xampp\htdocs\web\framework\web\filters\CFilter.php(40): CFilterChain->run()
#7 C:\xampp\htdocs\web\framework\web\CController.php(1145): CFilter->filter(Object(CFilterChain))
#8 C:\xampp\htdocs\web\framework\web\filters\CInlineFilter.php(58): CController->filterAccessControl(Object(CFilterChain))
#9 C:\xampp\htdocs\web\framework\web\filters\CFilterChain.php(130): CInlineFilter->filter(Object(CFilterChain))
#10 C:\xampp\htdocs\web\framework\web\CController.php(291): CFilterChain->run()
#11 C:\xampp\htdocs\web\framework\web\CController.php(265): CController->runActionWithFilters(Object(CInlineAction), Array)
#12 C:\xampp\htdocs\web\framework\web\CWebApplication.php(282): CController->run('create')
#13 C:\xampp\htdocs\web\framework\web\CWebApplication.php(141): CWebApplication->runController('handicap/create')
#14 C:\xampp\htdocs\web\framework\base\CApplication.php(185): CWebApplication->processRequest()
#15 C:\xampp\htdocs\web\index.php(13): CApplication->run()
#16 {main}
REQUEST_URI=/web/handicap/create
HTTP_REFERER=http://localhost/web/handicap/create
---
